[2025-08-06 19:29:55,571] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-06T17:29:30.234569+00:00 [queued]>
[2025-08-06 19:29:55,590] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-06T17:29:30.234569+00:00 [queued]>
[2025-08-06 19:29:55,590] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-06 19:29:55,590] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-06 19:29:55,590] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-06 19:29:55,611] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-06 17:29:30.234569+00:00
[2025-08-06 19:29:55,621] {standard_task_runner.py:52} INFO - Started process 3502 to run task
[2025-08-06 19:29:55,625] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'scheduled__2025-08-06T17:29:30.234569+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmph4drfy2h', '--error-file', '/tmp/tmp2779feh9']
[2025-08-06 19:29:55,626] {standard_task_runner.py:80} INFO - Job 3: Subtask verify_pipeline
[2025-08-06 19:29:55,700] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-06T17:29:30.234569+00:00 [running]> on host 23b45da21b8f
[2025-08-06 19:29:55,956] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-06T17:29:30.234569+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-06T17:29:30.234569+00:00
[2025-08-06 19:29:55,957] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-06 19:29:55,958] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo "⏳ Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-06 19:29:55,983] {subprocess.py:85} INFO - Output:
[2025-08-06 19:29:55,985] {subprocess.py:92} INFO - ⏳ Attente traitement Logstash...
[2025-08-06 19:30:26,025] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-06 19:30:26,081] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250806T172930, start_date=20250806T192955, end_date=20250806T193026
[2025-08-06 19:30:26,161] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-06 19:30:26,209] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
